test/util.ts

```diff
@@ -31,6 +31,13 @@ export function parseUnitModelWithScale(spec: TopLevel<UnitSpec>) {
   return model;
 }
 
+export function parseUnitModelWithScaleAndLayoutSize(spec: TopLevel<UnitSpec>) {
```

Comment:
I'm not a huge fan of all of these test util function. I think tests should be as explicit as possible. But it's beyond the scope of this pr.